src/plugins/discover/public/services/saved_searches/saved_searches_utils.test.ts

```diff
@@ -19,20 +17,6 @@ import { createSearchSourceMock } from '@kbn/data-plugin/public/mocks';
 import type { SavedSearchAttributes, SavedSearch } from './types';
 
 describe('saved_searches_utils', () => {
-  describe('getSavedSearchUrl', () => {
-    test('should return valid saved search url', () => {
-      expect(getSavedSearchUrl()).toBe('#/');
-      expect(getSavedSearchUrl('id')).toBe('#/view/id');
-    });
-  });
-
-  describe('getSavedSearchFullPathUrl', () => {
-    test('should return valid full path url', () => {
-      expect(getSavedSearchFullPathUrl()).toBe('/app/discover#/');
-      expect(getSavedSearchFullPathUrl('id')).toBe('/app/discover#/view/id');
-    });
-  });
```

Comment:
Moved to `src/plugins/discover/common/services/saved_searches/saved_searches_url.test.ts`